packages/webapp/webapp_server.js

```diff
@@ -266,30 +264,24 @@ var getBoilerplate = function (request, arch) {
     }
     return memoizedBoilerplate[memHash];
   }
-  
-  var boilerplateOptions = _.extend({ 
-    htmlAttributes: htmlAttributes 
+
+  var boilerplateOptions = _.extend({
+    htmlAttributes: htmlAttributes
   }, _.pick(request, 'dynamicHead', 'dynamicBody'));
```

Comment:
That was just my editor. Not sure I want to get into the selective ES2015-izing game.